src/2_mockFn.test.js

```diff
@@ -1,7 +1,22 @@
 // Your tests here...
 // You'll practice the basics of jest mocks
-import { guardPower } from './2_mockFn';
+import { guardPower } from "./2_mockFn";
 
-it('...', () => {
-  // ...
+describe("guardPower()", () => {
+  it("should return cb(power) if power is less than 500", () => {
```

Comment:
suggestion: In my tests I'm influenced by the ["given when then"](https://www.newthings.co/blog/structuring-tests-using-given-when-then/) terminology. So here, I'd say:

"given power less than 500, return cb(power".

This is just a personal preference, sometimes I don't follow it, and definitely I don't let it overcomplicate the reading of the tests. For example, I don't create deeply nested `describe()` fns as in the article, unless I have a lot of repetition. 

I don't have a bulletproof recommendation for you, the simplest recommendation is: use your gut, and keep things simple.